mne/utils/docs.py

```diff
@@ -24,10 +24,20 @@
 
 # Verbose
 docdict['verbose'] = """
-verbose : bool | str | int | None
-    If not None, override default verbose level (see :func:`mne.verbose`
-    and :ref:`Logging documentation <python:tut-logging>` for more).
-    If used, it should be passed as a keyword-argument only."""
+verbose : bool | 'debug' | 'info' | 'warning' | 'error' | 'critical' | int | None
+    Controls the verbosity of the generated logging output. Logging messages
+    below the specified level will not be emitted. For example, if you specify
+    ``verbose='warning'``, all ``debug`` and ``info`` messages will be hidden.
+
+    You may also pass a boolean, where ``True`` is an alias for ``'info'`` and
+    ``False`` is an alias for ``'warning'``. See also :func:`mne.verbose`.
+
+    If ``None``, use the default logging level.
+
+    Integers are accepted for consistency with the underlying Python logging
+    system.
+
```

Comment:
99% of the time I think users will want to use `True` or `False`. I'd rather just document these as the basic options, and for other options refer them to the logging documentation as we do now. Otherwise all of our docstrings get a lot longer, which means much longer API pages, etc. with very little gain for the end users.

Reply:
I also feel that nobody will want to pass integers here these days 😅